docs/source/conf.py

```diff
@@ -20,6 +20,7 @@
     "python": ("https://docs.python.org/3/", None),
 }
 
+autodoc_mock_imports = ["dnf", "yum", "rpm", "hawkey", "dbus", "pexpect"]
```

Comment:
Can we have a link to some docs that explain what the autodoc_mock do?

Reply:
@r0x0d Added!